api.ts

```diff
@@ -1108,6 +1110,25 @@ export interface SyncUpdateResultMarshaled<
  */
 export type GenericSyncUpdateResultMarshaled = SyncUpdateResultMarshaled<any, any, any>;
 
+/**
+ * Sets a limit on the number of permissions that can be returned for a single row
+ * in a call to {@link executeGetPermissions}.
+ *
+ * @hidden
+ */
+const MaxPermissionsPerRow = 1000;
```

Comment:
Good question. We thought this was a reasonable limit we can lift/change in the future.

I think at there are a two options I was thinking about for the future: 

1) Just limit the number of entities per batch instead of for a single item in a batch. That way someone can just change the batchSize if there are tons of permissions.
2) Add a continuation concept to the execution context/return if we hit this limit. If we see continuation we pass in the same rows until we do not see one.

Wanted to be fairly strict to start out and we can unwind this if we find the need